# Goal definition
geometry_msgs/PoseStamped[] pose_list
---
# Result definition
string status
---
# Feedback
int32 current_index
